vowpalwabbit/cb_explore.cc

```diff
@@ -54,6 +54,7 @@ struct cb_explore
   bool nounif = false;
   bool epsilon_decay = false;
   VW::version_struct model_file_version;
+  VW::io::logger* logger;
```

Comment:
other reductions get a copy to the `all.logger` object and others hold a pointer to it?

Reply:
I need to check this specific place but the logger object cannot be initialized, so either the constructor initializes it or it must be some optional value. Clearly optional would be better